plugins/modules/zypper.py

```diff
@@ -588,7 +614,7 @@ def main():
         elif state in ['installed', 'present', 'latest']:
             packages_changed, retvals = package_present(module, name, state == 'latest')
 
-    retvals['changed'] = retvals['rc'] in [0, 102] and bool(packages_changed)
+        retvals['changed'] = retvals['rc'] in [0, 102] and bool(packages_changed)
```

Comment:
Is this change intentional?

Reply:
This was not intentional, I am not sure why some of these spacing issues were added in as my git diffs don't show that. Will update this accordingly.

Reply:
Submitted another commit fixing this.